README.md

```diff
@@ -1,2 +1,73 @@
-# prometheus-opensearch-dashboards-exporter
+# Prometheus Opensearch Dashboards Exporter
 An exporter for OpenSearch Dashboards
+
+
+## Installing
+With [poetry](https://python-poetry.org/) installed, clone this repo and run:
+
+```shell
+poetry init
+```
+
+```shell
+poetry build
+```
+
+Builds will be located at `dist`. Access the virtual environment and install the build package.
+
+```shell
+poetry shell
+
+pip install ./dist/prometheus_opensearch_dashboards_exporter-0.1.0.tar.gz
+```
+
+
+## Running
+Identify the url of the `opensearch-dashboards` that you want to monitor, set the `OPENSEARCH_DASHBOARDS_USER` and `OPENSEARCH_DASHBOARDS_PASSWORD` and start the Prometheus exporter:
+
+```shell
+export OPENSEARCH_DASHBOARDS_USER="my-user"
+export OPENSEARCH_DASHBOARDS_PASSWORD="my-pass"
+prometheus-opensearch-dashboards-exporter  http(s)://<IP>:5601
```

Comment:
It's regarding security. It's not ideal to pass passwords in the CLI because any user could run `ps` and see the arguments used. When you use environmental variables, just the owner and root would have access to the environmental variables.

In the end of the day, this will be used in opensearch-dashboards-operator that is using a config file ([which does not have the right access yet](https://github.com/canonical/opensearch-dashboards-operator/issues/135)) and we'll read from it.

However this might be used by the community separately from the charm, and I would like to implement better practices.